### PR TITLE
create login page

### PR DESCRIPTION
**Pull request by irenelee-hll**

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Footer from '../components/Footer/Footer'
+import LoginForm from '../components/LoginForm/LoginForm'
+
+
+export default function Login() {
+    return (
+        <div>
+      <Header></Header>
+      <LoginForm></LoginForm>
+      <Footer></Footer>
```
Comment (ivanms1) [inline client/src/pages/Login.js]: `<Header />` `<LoginForm />` `<Footer />`

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import Footer from '../components/Footer/Footer'
+import SignUpForm from '../components/SignUpForm/SignUpForm'
+
+
+export default function SignUp() {
+    return (
+        <div>
+      <Header></Header>
+      <SignUpForm></SignUpForm>
+      <Footer></Footer>
```
Comment (ivanms1) [inline client/src/pages/SignUp.js]: Same as above

File: client/src/components/SignUpForm/SignUpForm.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            password: '',
+            email: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(e) {
```
Comment (Sosodope) [inline client/src/components/SignUpForm/SignUpForm.js:22]: use ES6 arrow functions to bind this
https://www.freecodecamp.org/news/this-is-why-we-need-to-bind-event-handlers-in-class-components-in-react-f7ea1a6f93eb/

File: client/src/pages/Login.js (hunk 1 above)
Comment (Sosodope) [reply client/src/pages/Login.js]: Good note, please update this Irene and I will try to do the same in my components

File: client/src/components/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            username: '',
+            password: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(e) {
```
Comment (Sosodope) [inline client/src/components/LoginForm/LoginForm.js:21]: use ES6 arrow functions to bind this
https://www.freecodecamp.org/news/this-is-why-we-need-to-bind-event-handlers-in-class-components-in-react-f7ea1a6f93eb/

File: client/src/components/Footer/Footer.js [hunk 5]
```diff
@@ -41,9 +41,9 @@ const Footer = () => {
           <div className="social-column list-content">
           <p>Social</p>
             <ul className="menu-wrapper">
-              <li className = "menu-item"><Link to="/">Facebook</Link></li>
-              <li className = "menu-item"><Link to="/">Twitter</Link></li>
-              <li className = "menu-item"><Link to="/">Dribble</Link></li>
+              <li className = "menu-item"><Link to="http://www.twitter.com/meetup"><i class="fab fa-twitter" ></i> Twitter</Link></li>
```
Comment (Sosodope) [inline client/src/components/Footer/Footer.js]: External links use `a` tags, Link tags are usually reserved for internal links

**Review by irenelee-hll — COMMENTED** [review]
changd

**Review by ivanms1 — APPROVED** [review]

**Review by Sosodope — COMMENTED** [review]
Switch to arrow functions for methods and replace Link tags with a tags for external links

File: client/src/components/Footer/Footer.js (hunk 5 above)
Comment (irenelee-hll) [reply client/src/components/Footer/Footer.js]: ok

**Pull request merged**